Opportunity hero

JamBai2812 opened the pull request:
-This branch was a changed ticket for 'Generic Hero Component'
-There is now one component that can be used for all three landing page hero images/titles called 'Hero'
-styling is all currently in the Hero.scss file, constants used can be replaced when actual themes get chosen. All three theme colour constants are currently set to grey.
-I've added preliminary images to the three components on the landing pages that can be replaced at a later date.

Commit:
created reusable hero component

Commit:
Added styling to reusable component

Commit:
Added new hero component to all three landing pages

Commit:
Added placeholder images for opportunity and spirit landing pages

MichaelWalker commented on src/Components/Hero/Hero.scss:
```diff
@@ -0,0 +1,65 @@
+﻿$curiosityThemeColor: grey;
+$spiritThemeColor: grey;
+$opportunityThemeColor: grey;
```
These seem like nice things to have in the shared constants

MichaelWalker replied:
although perhaps this hasn't been merged in yet?

JamBai2812 replied:
Yeah I'm waiting for the constants file to get merged into master and then can be swapped in

MichaelWalker commented on src/Components/Hero/Hero.scss:
```diff
@@ -0,0 +1,65 @@
+
+
+.hero {
+    //temp
```
instead of temp, a common pattern is to use `// TODO - can be removed once xxx`.

Most IDE's understand the word 'TODO' and make them really easy to spot / find across the whole project.
Its also a good idea to note down what it's blocked on, so that it can confidently be removed once that thing is done!

MichaelWalker commented on src/Components/Hero/Hero.tsx:
```diff
@@ -0,0 +1,27 @@
+
+export function Hero(props : HeroProps){
+
+    const heroClass : string = "hero " + props.rover;
```
Your version works just fine, but we can probably tidy this up a little by doing it all inline.

```jsx
<div className={`hero ${props.rover}`}>
```

JamBai2812 replied:
way better thanks

MichaelWalker reviewed (approved):
Looks good.

I've left a couple of suggestions that you can tweak if you like, but it looks like it should all work fine!

Commit:
Made suggested edits - see description

-changed temp comment to TODO, created props class inline rather than separate variable
